src/index.html

```diff
@@ -6,13 +6,64 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file (`index.scss`). Typically, browsers cannot directly interpret SCSS files. Ensure that this SCSS file is compiled to CSS and linked appropriately, or change the link to point to the compiled CSS file.